tests/interactions/focusable-test.js

```diff
@@ -15,11 +15,9 @@ describe('BigTest Interaction: focusable', () => {
   beforeEach(() => {
     focused = false;
 
-    document.querySelector('.test-input').addEventListener('focus', () => {
-      if (document.querySelector('.test-input') === document.activeElement) {
-        focused = true;
-      }
-    });
+    document
+      .querySelector('.test-input')
+      .addEventListener('focus', e => (focused = e.target === document.activeElement));
```

Comment:
Now I'm just being picky 😆 

We don't have to one-line the handler, I think the other way was a bit clearer:

```js
document.querySelector('.test-input').addEventListener('focus', e => {
   focused = e.target === document.activeElement;
});
```

I one-lined it in my previous comment just for brevity